src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net
+{   
+    internal partial class CertModule : CertInterface
+    {
+        #region internal Methods
+        internal override SslPolicyErrors VerifyRemoteCertName(X509Chain chain, bool isServer, string hostName)
+        {
+            SslPolicyErrors sslPolicyErrors = SslPolicyErrors.None;
+
+            unsafe
+            {
+                uint status = 0;
+
+                var eppStruct = new Interop.Crypt32.SSL_EXTRA_CERT_CHAIN_POLICY_PARA()
+                {
+                    cbSize = (uint)Marshal.SizeOf<Interop.Crypt32.SSL_EXTRA_CERT_CHAIN_POLICY_PARA>(),
+                    dwAuthType = isServer ? Interop.Crypt32.AuthType.AUTHTYPE_SERVER : Interop.Crypt32.AuthType.AUTHTYPE_CLIENT,
+                    fdwChecks = 0,
+                    pwszServerName = null
+                };
+
+                var cppStruct = new Interop.Crypt32.CERT_CHAIN_POLICY_PARA()
+                {
+                    cbSize = (uint)Marshal.SizeOf<Interop.Crypt32.CERT_CHAIN_POLICY_PARA>(),
+                    dwFlags = 0,
+                    pvExtraPolicyPara = &eppStruct
+                };
+
+                fixed (char* namePtr = hostName)
+                {
+                    eppStruct.pwszServerName = namePtr;
+                    cppStruct.dwFlags |= (Interop.Crypt32.CertChainPolicyIgnoreFlags.CERT_CHAIN_POLICY_IGNORE_ALL &
```

Comment:
Nit: alignment was lost during copy/paste. Please try to keep line length under 120 chars.